packages/sdk/contractkit/src/wrappers/Governance.ts

```diff
@@ -387,9 +387,7 @@ export class GovernanceWrapper extends BaseWrapper<Governance> {
   getApproverMultisig = () =>
     this.getApprover().then((address) => this.kit.contracts.getMultiSig(address))
 
-  async getProposalStage(proposalID: BigNumber.Value): Promise<ProposalStage> {
-    // NOTE: `methods.getProposalStage` is broken for case where proposal is queued and expired
-    // can be reverted once https://github.com/celo-org/celo-monorepo/pull/6502 is released
```

Comment:
this hasn't been released, as confirmed by `celocli network:contracts --node https://forno.celo.org --filter Contract=Governance`

```
Contract   Proxy                                      Implementation                             Version CELO      cUSD      cEUR
Governance 0xD533Ca259b330c7A88f74E000a3FaEa2d63B7972 0xe7A9d32D87910f4251c54587E447D6124Db6Fd37 1.2.0.2 5.522e+24 4.304e+19 2.000e+24
```